static/styles/portico-signin.scss

```diff
@@ -981,3 +981,36 @@ button.login-google-button {
     height: 45px;
     width: 325px;
 }
+
+#submit_button.inactive {
+    display: inline-block;
+    vertical-align: top;
+    padding: 15px 22px 13px 22px;
+    font-family: "Source Sans Pro";
+    font-size: 1.2rem;
+    font-weight: 400;
+    box-sizing: border-box;
+    outline: none;
+    color: #fff;
+    background-color: azure;
+    border: none;
+    transition: all 0.3s ease;
+}
+
+#submit_button.active {
```

Comment:
Find style applied to existing buttons with this type of style, i.e. submit button signup page, login page. And apply those same style `Find Account` to button instead of defining all again. It might be a case, that `Find Account` button require further style, in that case you can apply them this way.

Reply:
There wasn't styling declared or any relevant information on how the inactive button should look. So I designed a separate so that it can be changed by the designer

Reply:
Okay, but lets discuss the inactive button style on #design channel(start new thread and post screenshots) first, get feedback on UI.